contracts/clients/README.md

```diff
@@ -22,10 +22,29 @@ const netCfg: NetworkConfig = await getConfig(
 
 Exposes typed functions for interacting with the Aggregator's HTTP API.
 
+### Add a bundle to an aggregator
+
+```ts
+import { Aggregator } from "bls-wallet-clients";
+
+const aggregator = new Aggregator("https://arbitrum-goerli.blswallet.org");
+const resp = await aggregator.add(bundle); // See BlsWalletWrapper section below
+// Aggregator did not accept bundle
+if ("failures" in resp) {
+  throw new Error(resp.failures.join(", "));
+}
+```
+
+### Get the bundle receipt that contains the transaction hash you can lookup on a block explorer
+
+You will have to poll for the bundle receipt once you have added a bundle to an aggregator. The transaction hash is located on the bundle receipt. The property you need is `bundleReceipt.transactionHash`. This represents the transaction hash for the bundle submitted to the Verification Gatewaty, and can be used in a block explorer.
+
+Note this transaction is reprentative of the entire bundle submitted by the aggregator, and does not represent individual operations. To retrieve information about individual operations, use the get `getOperationResults` helper method which is explained under the [VerificationGateway](#verificationgateway) section below.
+
 ```ts
-import { Aggregator } from 'bls-wallet-clients';
+import { Aggregator } from "bls-wallet-clients";
 
-const aggregator = new Aggregator('https://arbitrum-goerli.blswallet.org');
+const aggregator = new Aggregator("https://arbitrum-goerli.blswallet.org");
```

Comment:
Outside the scope of these changes, but I think we need to be more L2 agnostic in our examples going forward. Something like:

```suggestion
const aggregator = new Aggregator("bls-wallet.aggregator.url")
```

Somewhat related is providing defaults for the ones we host: https://github.com/web3well/bls-wallet/issues/199

Reply:
yeah that makes sense, happy to amend as part of this change if we like `bls-wallet.aggregator.url`.

Reply:
Let's do as followup at some point, not high priority. Would rather get these updates in